src/_includes/navbar.html

```diff
@@ -1,9 +1,9 @@
 <header class="navbar">
   <ul aria-label="Main navigation">
     <li>
-      <a href="/"
-        ><img src="/assets/d-marque.png" role="presentation" alt="" /> Home</a
-      >
+      <a href="/" aria-label="Homepage">
+          <img src="/assets/dxw-logo.png" aria-hidden="true" alt="" />
```

Comment:
There shouldn't be a need for `aria-hidden="true"` here since an empty alt property indicates the image is decorative and therefore hidden from the accessibility tree of elements needing to be read out.